### DISCUSSION TITLE
forwarding to all of the chats the bot is in

### DISCUSSION THREAD

**Discussion by Pallenere555** [MrMissx/Telegram_Forwarder#37] [Q&A]
is it possible to make it forward to all of the chats the bot is in?

**Comment by MrMissx** [comment]
I think there is no option to make the bot know what chats are the bot in. So the only way is by putting all the chat_ids on the `TO_CHATS` variable

**Reply by Pallenere555** [reply]
ik but there are already some bots which do that so i tought it could be done here too

**Reply by MrMissx** [reply]
Yes, they use a database to store a chat update so they have the chat_id data to send the message.

**Reply by Pallenere555** [reply]
oh, so there is no way I could apply this to the bot?

**Reply by MrMissx** [reply]
Yes you can apply that. but not with the current code
